### ISSUE TITLE
Dependency Error on Installing `@graphprotocol/graph-cli` due to `ipfs-http-client@34.0.0`

### ISSUE THREAD

**Issue by mihirgupta0900** [graphprotocol/graph-tooling#1262]
### Which packages are impacted by your issue?

@graphprotocol/graph-cli

### Describe the issue

When attempting to install `@graphprotocol/graph-cli`, a dependency error occurs. This issue originates from the `ipfs-http-client@34.0.0` package.

The error stems from version `34.0.0` of the package, which attempts to install `concat-stream` from a [forked version](https://github.com/hugomrdias/concat-stream) on the `feat/smaller` branch. Unfortunately, this branch was deleted a few hours ago ([link to branch](https://github.com/hugomrdias/concat-stream/tree/feat/smaller)) ([link to PR](https://github.com/maxogden/concat-stream/pull/62)). Refer to the screenshot below for additional context:

![image](https://user-images.githubusercontent.com/37367148/230724552-c42b384d-8c34-48cd-bfbd-564abdfeb27c.png)

Upon inspecting the node_modules of the cli package, it is evident that concat-stream is being installed from the forked version:
```json
    "concat-stream": "github:hugomrdias/concat-stream#feat/smaller",
```
![image](https://user-images.githubusercontent.com/37367148/230724723-9074a6dd-08c0-43d1-adbc-a8e9e10a565b.png)

I believe upgrading the version for `ipfs-http-client` will fix this issue

### Reproduction

_Steps listed above_

### Steps to Reproduce the Bug or Issue

`pnpm i @graphprotocol/graph-cli`

### Expected behavior

To install

### Screenshots or Videos

_Attached above_

### Platform

- OS: macOS

**Comment by grmkris** [comment]
following this..

in my pnpm monorepo, i did this override..
```
"pnpm": {
    "overrides": {
      "ipfs-http-client": "60.0.0"
    }
  },
```

install goes through, but now i need to test the subgraphs if all still works

**Comment by mihirgupta0900** [comment]
> following this..
> 
> in my pnpm monorepo, i did this override..
> 
> ```
> "pnpm": {
>     "overrides": {
>       "ipfs-http-client": "60.0.0"
>     }
>   },
> ```
> 
> install goes through, but now i need to test the subgraphs if all still works
> 
> edit: didn't work..
> 
> but this probably will:
> 
> ```
>  "pnpm": {
>     "overrides": {
>       "ipfs-http-client@34.0.0>concat-stream": "github:mihirgupta0900/concat-stream#master"
>     }
>   }
> ```

Yes, that works for the time being 👍

**Comment by saihaj** [comment]
should be patched in [@graphprotocol/graph-cli@0.45.2](https://github.com/graphprotocol/graph-tooling/releases/tag/release-1681166300461)
